src/module-elasticsuite-core/Api/Search/Request/ContainerConfigurationInterface.php

```diff
@@ -74,4 +74,13 @@ public function getRelevanceConfig();
      * @return integer
      */
     public function getStoreId();
+
+    /**
+     * Retrieve filters for the container (visibility, in stock, etc ...) and the current search Context.
+     *
+     * @param \Smile\ElasticsuiteCore\Search\Context $searchContext Search Context
+     *
+     * @return \Smile\ElasticsuiteCore\Search\Request\QueryInterface[]
+     */
+    public function getFilters(\Smile\ElasticsuiteCore\Search\Context $searchContext);
```

Comment:
\Smile\ElasticsuiteCore\Search\Context should be an interface as it is used in the API area.